packages/lodestar/test/perf/chain/validation/attestation.test.ts

```diff
@@ -26,3 +27,46 @@ describe("validate gossip attestation", () => {
     });
   }
 });
+
+describe.only("test throw AttestationError vs object", () => {
+  itBench({
+    id: "AttestationError",
+    fn: async () => {
+      for (let i = 0; i < 1000; i++) {
+        try {
+          await throwAttestationError(true);
+        } catch (e) {}
+      }
+    }
+  });
+
+  itBench({
+    id: "Object",
+    fn: async () => {
+      for (let i = 0; i < 1000; i++) {
+        try {
+          await throwObject(true);
+        } catch (e) {}
+      }
+    }
+  });
+});
+
+async function throwAttestationError(b: boolean): Promise<void> {
+  if (b) {
+    throw new AttestationError(GossipAction.IGNORE, {
+      code: AttestationErrorCode.UNKNOWN_BEACON_BLOCK_ROOT,
+      root: Buffer.alloc(32),
```

Comment:
Don't allocate bytes here, this will distort the test's results. Declare this root above once for all tests or use ZERO_HASH constant